static/app/views/starfish/views/webServiceView/endpointOverview/index.tsx

```diff
@@ -351,6 +377,7 @@ const StyledRow = styled(PerformanceLayoutBodyRow)`
 
 const SegmentedControlContainer = styled('div')`
   margin-bottom: ${space(2)};
+  position: relative;
```

Comment:
if you want to align the other button to the right would it makes more sense to use flex?

Reply:
Yeah flex would likely be better